### PR TITLE
[Fix] Uninstallation of the RHOAI operator

### PR DESCRIPTION
**Pull request by jstourac**
Let's unify naming of the catalog and operator group to what we use
elsewhere.

Also removing the `embedded*.txt` files since these aren't used anywhere
in this repository anymore.

This follows up on #1273.

---

No CI run executed for this yet.

### CODE REVIEW

**Comment by jstourac** [comment]
Okay, I can see that most of the changes in this have been done meantime in #1273. Even though the commit message neither the PR description itself mentions anything about some fix.... :/

Gonna rebase.

File: ods_ci/tasks/Resources/RHODS_OLM/pre-tasks/oc_is_operator_installed.robot [hunk 1]
```diff
@@ -15,17 +15,17 @@ Is RHODS Installed
               ${result}=  Run Keyword And Return Status
               ...  Run Keywords
               ...  Check A RHODS Family Operator Is Installed  namespace=redhat-ods-operator
-              ...                                              subscription=rhods-operator-dev  AND
+              ...                                              subscription=rhoai-operator-dev  AND
```
Comment (jiridanek) [inline ods_ci/tasks/Resources/RHODS_OLM/pre-tasks/oc_is_operator_installed.robot:18]: what's the migration path if I install with old ods-ci, then pull this change, and then want to run ods-ci again?

(I'm trying to find something to complain and I just got notified from this PR, so, found something to complain about)

Comment (jstourac) [reply ods_ci/tasks/Resources/RHODS_OLM/pre-tasks/oc_is_operator_installed.robot:18]: there is no migration path from my side :smile:

Comment (aloganat) [reply ods_ci/tasks/Resources/RHODS_OLM/pre-tasks/oc_is_operator_installed.robot:18]: For the deployment with old CS name can be uninstalled manually and from then we can start using this.

**Review by jiridanek — APPROVED** [review]

**Review by aloganat — APPROVED** [review]

**Comment by jstourac** [comment]
Thank you, merged :slightly_smiling_face:

**Pull request merged**